website/templates/website/documentation_page.html

```diff
@@ -1,6 +1,15 @@
 {% extends "website/base_template.html" %}
 {% load customfilters %}
 
+{% block extra_css %}
+<style type="text/css">
+	.section {
+    padding-top: 50px;
```

Comment:
The problem is the permalinks jump to a div or section with the specific id. And I could not find any way to make it jump some pixels above that section. So the only way I could think of is pushing the contents of the sections down a bit. Then again how many pixels to shift has to be fixed. Lets see if there is a javascript solution.